content/harvester/_index.md

```diff
@@ -0,0 +1,28 @@
+# Testing CKAN harvesting of FAIR data points?
+
+Tutorial made on 20/12/2023
+
+We assume that CKAN is properly set up (as documented in this wiki) and running.
```

Comment:
`We assume that CKAN is properly set up and running, as documented in this wiki.` What do you think?